### DISCUSSION TITLE
Title should be configurable through openapi-merge.json

### DISCUSSION THREAD

**Discussion by santhosh-iouring** [iouring-engineering/openapi-merge#4] [General]
#Open API Title:
While merging multiple openapi 3.0 files using the tool @iouring-engineering/openapi-merge-cli, the title of the final output file is copied from the first input file mentioned in openapi-merge.json.  Title reference attached here,

![openapititle](https://user-images.githubusercontent.com/103554827/164391422-58be6c9c-104c-4e43-bd55-4ca9c0d39bda.JPG)

Since we are merging multiple micro service units to a single file, the title of the document will vary for each service stack
It should be really helpful if title can be configurable through openapi-merge.json.

**Comment by hari-iouring** [comment]
@santhosh-iouring - Could you add this as a requirement in the issue sections with label as enhancement. 

cc: @revanth-iouring

**Reply by santhosh-iouring** [reply]
@hari-iouring  Sure, will add this as a requirement

**Comment by revanth-iouring** [comment]
@santhosh-iouring thanks for bringing this requirement, will include this requirement and release new library.

cc @hari-iouring

**Comment by hari-iouring** [comment]
Discussion is closed. The requirement is raised as Issue and released a new version
